book/src/transaction.md

```diff
@@ -0,0 +1,70 @@
+# Anatomy of a Transaction
+
+This chapter documents the binary format of a transaction.
+
+## Transaction Format
+
+A transaction contains a compact-array of signatures, followed by a message.
+The signatures are all signatures of the given message. The Solana runtime
+verifies that the number of signatures matches the number in the first 8 bits
+of the message header. It also verifies the signature was signed by the
```

Comment:
```suggestion
of the message header. It also verifies that each signature was signed by the
```

Reply:
Mm, apparently github did preserve these comments on outdated lines. Sorry for the dupe!